maintainers/maintainer-list.nix

```diff
@@ -2249,6 +2249,11 @@
     github = "barinov274";
     githubId = 54442153;
   };
+  BarrOff = {
+    name = "BarrOff";
+    github = "BarrOff";
+    githubId = 58253563;
+  };
```

Comment:
e-mail, please

Reply:
I would prefer not to add my private mail to a public repo.
I checked the maintainer list and there are other entries consisting only of name, github, githubId so I thought this would be possible?.

Reply:
Usually I recommend a specialized e-mail for this.
But let's not pursue this now.